felles/src/test/kotlin/no/nav/helsearbeidsgiver/felles/InntektTest.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.helsearbeidsgiver.felles
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.time.YearMonth
+
+internal class InntektTest {
+
+    @Test
+    fun gjennomsnitt_Ingen_Inntekt_er_0() {
+        val inntekt = Inntekt(0.0, emptyList())
+        assertEquals(0.0, inntekt.gjennomsnitt())
+    }
+
+    @Test
+    fun gjennomsnitt_for_en_eller_ingen_maaned_er_lik_som_total() {
+        val total = 50000.0
+        val inntekt = Inntekt(total, emptyList()) // spesialtilfelle, liste bør aldri være tom, men håndter det likevel
+        assertEquals(total, inntekt.gjennomsnitt())
+        val inntekt2 = Inntekt(total, listOf(MottattHistoriskInntekt(YearMonth.now(), total)))
+        assertEquals(total, inntekt2.gjennomsnitt())
+    }
+
+    @Test
+    fun gjennomsnitt_for_to_maaneder() {
+        val inntekter = mapOf(1 to 10.0, 2 to 20.0)
+        val total = inntekter.values.sum()
+        val inntekt = Inntekt(total, genererHistoriskInntekt(inntekter))
+        val forventetSnitt = total / inntekter.size
+        assertEquals(forventetSnitt, inntekt.bruttoInntekt)
+    }
+
+    @Test
+    fun test_float_feil() {
+        val inntekter = mutableMapOf<Int, Double>()
+        for (x in 1..11) {
+            inntekter.put(x, 0.2)
+        }
```

Comment:
Jeg prøver å unngå alt som er mutable for alt det er verdt. Her trenger du vel egentlig en liste (ref. kommentar på `genererHistoriskInntekt`) og da kan du bruke `List(10) { 0.2 }`.

Reply:
Nå tror jeg at jeg skjønner spm over.. :) 
Det vil nok bli aktuelt å bruke nøkkel, mtp at vi sikkert vil teste med inntekt ala "2020-04": 200, "2020-02":100 osv, så en liste vil ikke gi oss den fleksibiliteten vi kanskje trenger. 
Men sikkert mulig å se på bruken av mutable, tar en titt